### PR TITLE
feat: return breezy widget to careers page

### PR DESCRIPTION
**Pull request by lamartire**

### CODE REVIEW

**Comment by lamartire** [comment]
@ifiri ready for review

File: pages/careers.vue [hunk 1]
```diff
@@ -128,7 +128,7 @@
         <template slot="title">
           Here are some of our current openings:
         </template>
-        <v-table>
+        <!--<v-table>
```
Comment (ifiri) [inline pages/careers.vue]: Why you comment code in layout and remove code from script? If you want to save it for future, maybe will be better comment scripts as well?

Comment (lamartire) [reply pages/careers.vue]: Yes, have saved it for future usage

Comment (ifiri) [reply pages/careers.vue]: I ask this because commented code in html will be in source code, that's not very good

Comment (kazurus) [reply pages/careers.vue]: @ifiri we should fix it in near future, it's ok

**Comment by dudiq** [comment]
seems something goes wrong...
- open console
- open home page
- on console you see the error
- open carrers page
- and no any list of positions

- but if reload carrers page, all is fine

image

**Comment by lamartire** [comment]
@dudiq problem not on the our side, we can't do something with that

**Comment by dudiq** [comment]
@lamartire maybe we can load breezy only on careers page, like [here](https://vueschool.io/articles/vuejs-tutorials/how-to-load-third-party-scripts-in-nuxt-js/) ?
but not for the whole page

**Comment by lamartire** [comment]
@dudiq that just do not work, I have tried

**Comment by dudiq** [comment]
@kazurus need to create another issue with breezy loading, doh...
but for return breezy widget is approved

**Comment by lamartire** [comment]
All done here, @dudiq, @kazurus

**Comment by dudiq** [comment]
@kazurus approved

**Pull request merged**